crates/bevy_pbr/src/render/pbr_functions.wgsl

```diff
@@ -125,14 +125,14 @@ fn alpha_discard(material: pbr_types::StandardMaterial, output_color: vec4<f32>)
 // in use (mesh vs. meshlet).
 fn sample_texture(
     texture: texture_2d<f32>,
-    samp: sampler,
```

Comment:
does this shader even compile anymore

Reply:
I changed all instances of the spelling mistake, so I would expect it to be semantically equivalent. I would also expect that compilation errors are caught in CI. If not, maybe we should add that.